Foundation/Decimal.swift

```diff
@@ -101,31 +101,31 @@ public struct Decimal {
 
 extension Decimal {
     public static let leastFiniteMagnitude = Decimal(
-        _exponent: 127,
+        _exponent: Int32(Int8.max),
```

Comment:
What do you think of such expression change?

Reply:
What's going on here, and where did this property come from? "Magnitudes" are always non-negative, but this is a huge negative number. I don't know why this property even exists, but the name and documentation are *extremely* misleading.

(@takasek This is tangential to your change, but this is the first I've seen this).